recipes/soci/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

Comment:
```suggestion
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

Reply:
With `set_property` the standard could decay if not supported by compiler, but it is required at least `cxx_std_11`.
Are you sure about this change @intelligide ?

Reply:
Your conanfile already checks that the compiler supports C++11. ;)